perf/intkey_workload/src/main.rs

```diff
@@ -209,6 +218,11 @@ fn run_load_command(args: &ArgMatches) -> Result<(), Box<Error>> {
             String::from("unsatisfiable must be a positive float in the range [0.0, 1.0]")
         })?;
 
+    let unnecessary: f32 = args.value_of("unnecessary")
+        .unwrap_or("0.0")
+        .parse()
```

Comment:
What happens if this is a negative float or bigger than 1?

Reply:
Less than zero will act like 0, and greater than 1 will act like 1, but I should put in some parsing so that something unexpected doesn't happen for the user.